src/components/utils/dnd-table/dnd-table.jsx

```diff
@@ -38,7 +38,7 @@ import ChipItemsInput from '../rhf-inputs/chip-items-input';
 export const MAX_ROWS_NUMBER = 100;
 const styles = {
     columnsStyle: {
-        display: 'flex',
+        display: 'inline-flex',
```

Comment:
This will have an impact at different level of the application because this component is quite used, are you sure it is necessary ?

Reply:
Yes, it's necessary because it fixes the component everywhere, such as in the parameters of 'voltage profile initialization' in the Voltage Limits tab. It feels like the tooltip is on the left side of the column, but it actually belongs to the column on the left

Reply:
ok